history.py

```diff
@@ -0,0 +1,28 @@
+from catalog import models as cmod
+
+class LastFiveMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+        # One-time configuration and initialization.
+
+    def __call__(self, request):
+
+        recent_list= request.session.get('last_five',[])
```

Comment:
Do you have a standard for which word combinations are camel case, and which are seperated_with_underscores? Just trying to understand for better context / readability.

Reply:
well this was the format it said online, i was just following that.